src/engine/enginemaster.cpp

```diff
@@ -443,21 +448,27 @@ void EngineMaster::process(const int iBufferSize) {
     }
 
     if (masterEnabled) {
-        // Mix the three channels together. We already mixed the busses together
-        // with the channel gains and overall master gain.
-        if (!m_pMasterTalkoverMix->toBool()) {
-            // Add Talkover to Master output
-            SampleUtil::copy4WithGain(m_pMaster,
-                    m_pOutputBusBuffers[EngineChannel::LEFT], 1.0,
-                    m_pOutputBusBuffers[EngineChannel::CENTER], 1.0,
-                    m_pOutputBusBuffers[EngineChannel::RIGHT], 1.0,
+        // Mix the three crossfader bus orientation buffers together
+        SampleUtil::copy3WithGain(m_pMaster,
+            m_pOutputBusBuffers[EngineChannel::LEFT], 1.0,
+            m_pOutputBusBuffers[EngineChannel::CENTER], 1.0,
+            m_pOutputBusBuffers[EngineChannel::RIGHT], 1.0,
+            iBufferSize);
+
+        if (m_pBoothTalkoverMix->toBool()) {
+            // Include the talkover mix in the booth output
+            SampleUtil::copy2WithGain(m_pMaster,
+                    m_pMaster, m_pTalkoverDucking->getGain(iBufferSize / 2),
                     m_pTalkover, 1.0,
                     iBufferSize);
+            SampleUtil::copy(m_pBooth, m_pMaster, iBufferSize);
```

Comment:
we can get rid of this copy loop, if we redirect booth to master. It would also be nice, if we are able to skip both processing in case it is not used.